lib/travis/api/v3/queries/request.rb

```diff
@@ -22,7 +22,11 @@ def schedule(repository, user)
         config:     config || {}
       }
 
-      perform_async(:build_request, type: 'api'.freeze, credentials: { token: token }, payload: JSON.dump(payload))
```

Comment:
Is `:build_request` the queue name? `Travis::Sidekiq.gatekeeper` enqueues into `build_requests`.
I vaguely remember we have two queues: `build_request` and `build_requests`.
Could you confirm?